Emby.Server.Implementations/Data/BaseSqliteRepository.cs

```diff
@@ -25,88 +23,45 @@ protected BaseSqliteRepository(ILogger logger)
 
         protected TransactionMode ReadTransactionMode => TransactionMode.Deferred;
 
-        internal static int ThreadSafeMode { get; set; }
-
         protected virtual ConnectionFlags DefaultConnectionFlags => ConnectionFlags.SharedCached | ConnectionFlags.NoMutex;
 
-        private readonly SemaphoreSlim WriteLock = new SemaphoreSlim(1, 1);
-
-        private SQLiteDatabaseConnection WriteConnection;
+        private readonly SemaphoreSlim _writeLock = new SemaphoreSlim(1, 1);
 
-        private readonly BlockingCollection<SQLiteDatabaseConnection> ReadConnectionPool = new BlockingCollection<SQLiteDatabaseConnection>();
-
-        static BaseSqliteRepository()
-        {
-            ThreadSafeMode = raw.sqlite3_threadsafe();
-            raw.sqlite3_enable_shared_cache(1);
-        }
+        private SQLiteDatabaseConnection _writeConnection;
 
         private string _defaultWal;
 
-        protected async Task CreateConnections()
+        protected ManagedConnection GetConnection(bool _ = false)
```

Comment:
why are you leaving `_` parameter here?

Reply:
Didn’t want to break all uses in this PR.